.azure-devops/templates/compliance-job.yml

```diff
@@ -0,0 +1,58 @@
+# These steps have to run on a windows machine,
+# and therefore unfortunately can't be integrated in the regular steps
+
+jobs:
+  - job: compliance
+    displayName: Compliance checks
+    pool:
+      name: 1ES-Teams-Win2019-Security
+    steps:
+      - task: UseDotNet@2
+        condition: succeededOrFailed()
+        displayName: "Use .NET Core sdk 3.x"
+        inputs:
+          version: 3.x
+          steps:
+
+      - task: skvso.servicetree-build-tasks.servicetree-link-build-task.servicetree-link-build-task@1
+        displayName: "ServiceTree Integration"
+        inputs:
+          ServiceTreeGateway: "ServiceTree Gateway"
+          Service: 6F8CD842-E117-412F-BAE4-56A3B6166594
```

Comment:
Assuming this is graphitation's ID in the service tree? If this config is meant to serve as a template it would be good to call that out or change it to a variable defined at the start of the config?

Reply:
I think the right thing to do is to add a parameter to the template... lemme add that - this is a copy-pasta from fluentui :)

Reply:
OKAY this is fixed!